maat-court-data-api/src/main/java/gov/uk/courtdata/eform/dto/EformStagingDTO.java

```diff
@@ -14,4 +14,6 @@ public class EformStagingDTO {
     private Integer usn;
```

Comment:
The 2 new attributes should be in EformStagingResponse aswell?

Reply:
Have added `userCreated` to `EformStagingResponse` although won't include the `xmlDoc` as embedding escaped XML into a JSON response format is likely to break consumers or otherwise potentially cause downstream parsing issues.